PhpAmqpLib/Connection/AMQPSSLConnection.php

```diff
@@ -28,9 +28,11 @@ public function __construct(
         ?AMQPConnectionConfig $config = null
     ) {
         if (empty($ssl_options)) {
-            $ssl_options = ['verify_peer' => true];
+            trigger_error('Non-TLS instances of AMQPSSLConnection will be deprecated in version 4.0.0', E_USER_DEPRECATED);
```

Comment:
```suggestion
            trigger_error('Using non-TLS instances of AMQPSSLConnection is deprecated and won't we supported in php-amqplib/php-amqplib 4.0.0', E_USER_DEPRECATED);
```

Current message states that it will be deprecated since 4.0, but it is deprecated from now and probably will be dropped in 4.0.

Reply:
Thanks for the suggestion!